app/views/sep/approval/_admin_detail.html.erb

```diff
@@ -83,18 +84,60 @@
 						    <%= text_field_tag :option3_date, "", :'data-date-max'=> sep_optional_date(@family, 'max'), :'data-date-min'=> sep_optional_date(@family, 'min'),class: "floatlabel required date-picker form-control", placeholder: "DATE 3 OPTION" %>
 						</div>
 					</div>
+          <% if ::EnrollRegistry.feature_enabled?(:prior_plan_year_ivl_sep) %>
+            <div class='row row-form-wrapper no-buffer'>
+              <div class="col-md-12 col-sm-12 col-xs-12 form-group form-group-lg no-pd">
+                <%= check_box_tag :coverage_renewal_flag, 'false' , false, id: "coverage_renewal_flag", class: 'coverage_renewal_checkbox' %>
```

Comment:
coverage_renewal_flag need to display based on some conditions,

For example if user reporting sep today, we don't need to display this checkbox becoz their is nothing to renew now.

If we are in OE period or if admin reporting sep in prior plan year we need to display this checkbox.

By default this checkbox need to be true.